tests/unitTests/ArticleReducer.test.jsx

```diff
@@ -0,0 +1,32 @@
+import articleReducer from "../../src/reducers/articleReducer";
+import { POST_ARTICLE } from "../../src/actions/types";
+
+describe("socialLoginReducer", () => {
+  it("has a default state", () => {
+    expect(articleReducer(undefined, { type: "unexpected" })).toEqual({
+      article: {},
+      articles: []
+    });
+  });
+
+  it("updates state on POST_ARTICLE action type", () => {
+    expect(
```

Comment:
Could you clarify what the test is doing exactly

Reply:
@shakirandagire I have worked on this